### PR TITLE
Fix OBS build

### LINKED ISSUE #1: Missing libsamplerate dep
The specfile is missing `BuildRequires:	libsamplerate-devel` at the very least, I also had to add `BuildRequires:	gcc-c++` in my minimal docker env. I didn't figure out how to build without aac being an issue in configure so I'm no ready to do a PR yet.

### PR DESCRIPTION
**Pull request by hairmare**
Activate and install fdk-aac per default since the build process
doesn't have an easy way to deactivate this. This doesn't fix
disabling of aac in the builds but we have enough of nux-dextop
available right now for this to be pull-offable.

Fixes #1

### CODE REVIEW

File: butt.spec [hunk 1]
```diff
@@ -53,6 +53,7 @@ Source0:        http://downloads.sourceforge.net/%{name}/%{name}-%{version}.tar.
 BuildRequires:  desktop-file-utils
 BuildRequires:  fltk-devel
 BuildRequires:  portaudio-devel
+BuildRequires:  libsamplerate-devel
```
Comment (paraenggu) [inline butt.spec]: Please keep an alphabetical order here, thanks.

File: butt.spec [hunk 2]
```diff
@@ -23,8 +23,8 @@
 #
 
 # Conditional build support
-# add --with aac option, i.e. disable AAC by default
-%bcond_with aac
+# add --without aac option, i.e. enable AAC by default
```
Comment (paraenggu) [inline butt.spec:26]: What's the reason for this change? I don't quite understand to what build process you refer to with 

> Activate and install fdk-aac per default since the build process doesn't have an easy way to deactivate this.

**Review by paraenggu — CHANGES_REQUESTED** [review]

File: butt.spec (hunk 2 above)
Comment (hairmare) [reply butt.spec:26]: The `./configure` file checks for fdk-aac in any case and fails the build. I didn't find an obvious way to compile butt without aac support. I don't believe that the `--without` flags for other codecs even work (but didn't do any further testing). I did quite some hacking but coudn't figure out how to do a source build of butt without fdk-aac.

File: butt.spec (hunk 1 above)
Comment (hairmare) [reply butt.spec]: Done

File: butt.spec (hunk 2 above)
Comment (paraenggu) [reply butt.spec:26]: Ah ok, thanks for testing then! Actually, the butt build script doesn't have a way to explicitly enable or disable the specific codecs (it lacks any `--enable-` / `--disable-` or `--with-` / `--without-` options), it simply checks if the libraries are available and enables them if found. I wasn't aware that it doesn't work for AAC (I probably had the libs already installed while developing the spec file).

File: butt.spec (hunk 1 above)
Comment (paraenggu) [reply butt.spec]: Thanks!

**Comment by paraenggu** [comment]
@hairmare: Thank you! OBS build status: _succeeded_ - yeah!
